core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -667,6 +667,14 @@ public MetricKey build() {
           .setDescription("The last raft log index which was applied to the state machine")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey MASTER_JOURNAL_CHECKPOINT_WARN =
+          new Builder("Master.JournalCheckpointWarn")
+         .setDescription("If the raft log index exceeds "
+                  + "MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES, and the last checkpoint "
```

Comment:
Keep the indent consistent with  original style, just reference L687

Reply:
-> 

.setDescription(String.format("If the raft log index exceeds %s, and the last checkpoint exceeds %s, "
  + "it returns 1 to indicate that a warning is required, otherwise it returns 0", PropertyKey. MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES.getName(), PropertyKey. MASTER_WEB_JOURNAL_CHECKPOINT_WARNING_THRESHOLD_TIME.getName())